ProcessMaker/processmaker
# FOUR-11406: Stay in the same page when making a copy of an asset

## Issue & Reproduction Steps


When performing a copy operation from one project to another, the current behavior redirects the user to the asset that was copied. This disrupts the workflow and creates confusion.

1. Have a Project with multiple assets. (Process, Screen, Decision Table, Collection)
2. Navigate to the Project page.
3. Click on the Ellipsis menu of one of the Assets
4. Click on Copy to Project

### Actual behavior:
Users are redirected to the asset that has been copied, causing a disruption in the workflow.

### Expected behavior:
The user should stay within the same Project and experience a seamless copy operation. Additionally, the project page should reload to reflect the newly copied asset if applicable.

## Solution
- Modified the functionality to ensure that users remain in the same Project during the asset copy process.
- Implemented a page reload for the Project to reflect the changes made.

## How to Test
1. Set up a test environment with a Project containing multiple assets.
2. Navigate to the Project page.
3. Perform the copy operation using the steps provided.
4. Verify that you remain within the same Project and that the asset has been successfully copied to the intended target Project.

## Related Tickets & Packages
-  Ticket [FOUR-11406](https://processmaker.atlassian.net/browse/FOUR-11406)

## CI
ci:next
ci:package-projects:observation/FOUR-11343
ci:deploy

## Code Review Checklist


[FOUR-11406]: https://processmaker.atlassian.net/browse/FOUR-11406?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
